fs_types.go

```diff
@@ -120,7 +120,7 @@ type FSModify struct {
 	SmbNotifyOnChangeDirDepth  int32         `json:"smb_notify_on_change_dir_depth,omitempty"`
 	IsSmbNoNotifyEnabled       *bool         `json:"is_smb_no_notify_enabled,omitempty"`
 	IsAsyncMtimeEnabled        *bool         `json:"is_async_MTime_enabled,omitempty"`
-	ProtectionPolicyID         string        `json:"protection_policy_id,omitempty"`
+	ProtectionPolicyID         *string       `json:"protection_policy_id,omitempty"`
```

Comment:
May I know why are we changing the type to string pointer for these fields? To be assigned as nil? Isn't setting empty string not working anymore?

Reply:
Setting an empty string for these fields is being ignored because an empty string is considered a zero value for string variables.